src/data/nat/hyperoperation.lean

```diff
@@ -0,0 +1,127 @@
+/-
+Copyright (c) 2023 Mark Andrew Gerads. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mark Andrew Gerads, Junyan Xu, Alex J. Best, Eric Wieser, Patrick Johnson
```

Comment:
```suggestion
Authors: Mark Andrew Gerads, Junyan Xu, Alex J. Best, Eric Wieser
```

I'd prefer not to have my name associated with any kind of non-public-domain license. Besides that, my contribution to this file is so small that I should not be considered an author.

Reply:
I removed your name, as well as another that was a minor contributor. Can I have this hyperoperator file under the public domain even though all the other files seems to be under Apache 2.0?